proto/qdrant/cloud/cluster/v1/cluster.proto

```diff
@@ -468,6 +481,18 @@ message ClusterConfiguration {
       string: {ipv4_prefix: true}
     }
   }];
+  // The type of service to use for this cluster in a hybrid cloud environment.
+  // It is ignored for managed cloud clusters. This is an optional field, default is Cluster IP.
+  optional ClusterServiceType service_type = 12 [(buf.validate.field).enum = {
+    defined_only: true
+    not_in: [0]
+  }];
+  // List of annotations applied to the service of this cluster in a hybrid cloud environment.
+  // It is ignored for managed cloud clusters. This is an optional field
+  repeated common.v1.KeyValue service_annotations = 13 [(buf.validate.field).repeated = {max_items: 10}];
+  // List of labels applied to the pods of this cluster in a hybrid cloud environment.
+  // It is ignored for managed cloud clusters. This is an optional field
+  repeated common.v1.KeyValue pod_labels = 14 [(buf.validate.field).repeated = {max_items: 10}];
```

Comment:
we usually try to make the field names agnostic, however in this case I don't think it's possible. The product already refers to Kubernetes entities and we already have other fields (like tolerations) that refer to the Kubernetes documentation.

Reply:
Agreed